src/Resource/AbstractModel.php

```diff
@@ -56,16 +56,4 @@ public static function fromArray(array $data)
     {
         return new static($data);
     }
-
-    /**
-    * @ignore
-    * @codeCoverageIgnore
-    */
-    public function __get($name)
```

Comment:
What was this for?

Reply:
Hacky approach to access to protected fields.